docs/pyproject.md

```diff
@@ -395,6 +395,12 @@ You can install all extras with the `--all-extras` option:
 poetry install --all-extras
 ```
 
+Note that `install --extras` and the variations mentioned above (`--all-extras`, `--extras foo`, etc.) only work on dependencies defined in the current project. If you want to install extras defined by dependencies, you'll have to express that in the dependency itself:
+```toml
+[tool.poetry.group.dev.dependencies]
+fastapi = {version="^0.92.0", extras=["all"]}
+```
+
```

Comment:
**Would not be better putting that note as a `note` using `{{% note %}}`?**

```suggestion
{{% note %}}
Note that `install --extras` and the variations mentioned above (`--all-extras`, `--extras foo`, etc.) only work on dependencies defined in the current project. If you want to install extras defined by dependencies, you'll have to express that in the dependency itself:
```toml
[tool.poetry.group.dev.dependencies]
fastapi = {version="^0.92.0", extras=["all"]}
...
{{% /note %}}
```

Reply:
Done in latest commit - good suggestion, thanks @PabloEmidio